### PR TITLE
Initial commit for new PF threshold DB object

### PR DESCRIPTION
**Pull request by argiro**
Migration of Ecal PF thresholds from python-based to DB - based

### CODE REVIEW

**Comment by lpernie** [comment]
please test

**Comment by lpernie** [comment]
+1

**Comment by ggovi** [comment]
+1

**Comment by fabiocos** [comment]
+1

File: CondTools/Ecal/test/BuildFile.xml [hunk 1]
```diff
@@ -2,24 +2,4 @@
...
-  <test name="EcalTPGLutGroup_O2O_test" command="EcalTPGLutGroup_O2O_test.sh"/>
-  <test name="EcalTPGLinearizationConst_O2O_test" command="EcalTPGLinearizationConst_O2O_test.sh"/>
-  <test name="EcalTPGFineGrainTowerEE_O2O_test" command="EcalTPGFineGrainTowerEE_O2O_test.sh"/>
-  <test name="EcalTPGFineGrainStripEE_O2O_test" command="EcalTPGFineGrainStripEE_O2O_test.sh"/>
-  <test name="EcalTPGFineGrainEBIdMap_O2O_test" command="EcalTPGFineGrainEBIdMap_O2O_test.sh"/>
-  <test name="EcalTPGFineGrainEBGroup_O2O_test" command="EcalTPGFineGrainEBGroup_O2O_test.sh"/>
-  <test name="EcalTPGCrystalStatus_O2O_test" command="EcalTPGCrystalStatus_O2O_test.sh"/>
-  <test name="EcalTPGTowerStatus_O2O_test" command="EcalTPGTowerStatus_O2O_test.sh"/>
-  <test name="EcalADCToGeV_update_test" command="EcalADCToGeV_update_test.sh"/>
-  <test name="EcalIntercalibConstants_O2O_test" command="EcalIntercalibConstants_O2O_test.sh"/>
```
Comment (smuzaffar) [inline CondTools/Ecal/test/BuildFile.xml]: @ggovi , is this removal of tests on purpose?

**Comment by argiro** [comment]
Hi,

the removal was probably unintentional, but not critical ..

File: CondTools/Ecal/test/BuildFile.xml (hunk 1 above)
Comment (ggovi) [reply CondTools/Ecal/test/BuildFile.xml]: @smuzaffar thanks a lot to spot this!! 
@argiro please restore the tests, we are relying on these tests for the production O2Os = VERY CRITICAL

**Comment by ggovi** [comment]
hold

**Comment by ggovi** [comment]
@argiro probably a rebase would be a good option

File: CondTools/Ecal/test/BuildFile.xml (hunk 1 above)
Comment (smuzaffar) [reply CondTools/Ecal/test/BuildFile.xml]: @ggovi , I had made a PR https://github.com/cms-sw/cmssw/pull/22384 to restore these.

**Comment by smuzaffar** [comment]
@ggovi , this PR has already been merged, so no point putting a hold on it :-)
I have made a PR https://github.com/cms-sw/cmssw/pull/22384 to get these tests back

**Comment by argiro** [comment]
thanks !

**Comment by ggovi** [comment]
@argiro 
this PR has reverted also CondTools/Ecal/python/updateADCToGeV_test.py
CondTools/Ecal/python/updateIntercali_test.py

**Comment by ggovi** [comment]
@argiro please check again if more changes are unintended

**Comment by argiro** [comment]
I am asking  Francesca, who's the real author of the code ..

https://github.com/cms-sw/cmssw/pull/22346#issuecomment-369611795

**Comment by argiro** [comment]
sorry, it must have been a mistake.
Please leave the original version.
thanks
Francesca

**Pull request merged**